src/presets.rs

```diff
@@ -539,13 +575,28 @@ mod tests_gmp {
                 "difference between GMP and us: GMP reports {expected}, we report {actual}",
             );
 
-            let actual = fips::is_prime(&mut rng, Flavor::Any, p.as_ref(), mr_iterations, false, false);
+            let actual = fips::is_prime(
+                &mut rng,
+                Flavor::Any,
+                p.as_ref(),
+                mr_iterations,
+                fips::FipsOptions::default(),
```

Comment:
I wonder, does FIPS specify or recommend a specific number of m-r iterations? If yes, it'd be natural to have that inside  `FipsOptions` (and thus opening the nasty can of worms of which arguments go in an options struct and which don't).

Reply:
Included the MR iterations into `FipsOptions`. 

One thing I'm not sure about: technically, `FipsOptions::with_mr_iterations()` is hazmat, but I imagine one would want to cache the number of iterations since it's not trivial to calculate. A super safe approach is to make `minimum_mr_iterations` return a special newtype around `usize`, which `FipsOptions::with_mr_iterations()` takes. Too much?